graftm/hmmer.py

```diff
@@ -26,57 +42,54 @@ def __init__(self, search_hmm, aln_hmm=None):
         self.aln_hmm = aln_hmm
         self.hk = HouseKeeping()
 
-    def hmmalign(self, input_path, run_stats, for_file, rev_file, for_conv_file, rev_conv_file):
+    def hmmalign(self, input_path, directions):
         '''
-        hmmalign Align reads to the aln_hmm. Receives unaligned sequences and 
+        hmmalign - Align reads to the aln_hmm. Receives unaligned sequences and 
         aligns them.
         
         Parameters
         ----------
         input_path : str
             Filename of unaligned hits to be aligned
-        run_stats : dict
-            Dictionary containing run stats. Will have run time from alignment 
-            added to it. Also tells us if there are reads in the reverse 
-            compliment if in nucleotide pipeline
-        for_file : str
-            Output forward compliment path.
-        rev_file : str
-            Output reverse compliment path.
-        for_conv_file : str
-            Output alignment of forward reads with insertions removed
-        rev_conv_file : str
-            Output alignment of reverse reads with insertions removed
-        
+        directions : dict
+            dictionary containing readnames as keys, and compliment direction
```

Comment:
why not
```
any([direction == False for direction in directions.values()])
```
though? Maybe this is too hard though as you'd want to change the HMM table parser, which I think you use outside graftM now.